### ISSUE TITLE
Feature request: ignore localhost

### ISSUE THREAD

**Issue by hannesherold** [Daandelange/kirby3-simplestats#18]
For development, it would be very helpful if the `localhost` environment was ignored. 

For example, via a setting in `plugins/simplestats/src/config/options.php`:
`tracking => ignore => localhost => true (default)`.

In this context, it would also be useful if certain data could be deleted manually (for example, `localhost`).

**Comment by Daandelange** [comment]
Out of curiosity, why would you like to disable tracking on localhost ? (*maybe I'm unaware of certain dev workflows*)
On my localhost pc, I'm using a different db file than online (different server), and I also like to know that (a different) tracking works correctly, for testing purposes.
Maybe to ensure a local db (backup) doesn't get modified, but that it is still locally viewable trough the panel ?
Should "not-tracking-localhost" be enabled by default as you suggest ?

Would this condition work ?
````php
if( in_array($_SERVER['REMOTE_ADDR'], array('127.0.0.1', '::1')) ){ /* ... ignore ... */ }
````

As for data deletion, depending on what you mean, it will not be possible.  
- Erasing UserAgents / referrers : maybe (limited).
- Erasing visit / device related data related to localhost : no (there's no way to know).

**Comment by Daandelange** [comment]
I have implemented something in 01914d6053e65450b4e2b9fd279e5f9565fd3dd9
It looks like it's working; feel free to re-open otherwise.
It's not enabled by default (not-to-break stuff), that can still be discussed.

**Comment by hannesherold** [comment]
Thank you for addressing the issue! As an explanation of why I thought it made sense: In simple dev workflows, the entire public folder is uploaded via FTP - including the database.

Now that I think about it, it doesn't make much sense. Because once the website is online, the live database will no longer be replaced. So you're absolutely right, my problem won't even occur because you generally work locally with a different database than live.

Please excuse me for giving you unnecessary work!
